specs/001-swipe-gallery/data-model.md

```
@@ -0,0 +1,498 @@
+    var deletionCount: Int32 { get }
+    var compressionCount: Int32 { get }
+
+    // MARK: - In-Memory Index (O(1) lookups)
+    var decisionIndex: [String: SwipeDecision] { get }
+}
+```
+
+### Storage Optimization (Smallest Types)
+
+| Field | Type | Size | Reason |
+|-------|------|------|--------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)